### ISSUE TITLE
Documentation indicates null_ls.helpers exists

### ISSUE THREAD

**Issue by euoia** [jose-elias-alvarez/null-ls.nvim#129]
### FAQ

- [X] I have checked [FAQ](https://github.com/jose-elias-alvarez/null-ls.nvim#faq) and it didn't resolve my problem.

### Issues

- [X] I have checked [existing issues](https://github.com/jose-elias-alvarez/null-ls.nvim/issues?q=is%3Aissue) and there are no issues with the same problem.

### Neovim Version

NVIM v0.5.0

### Steps to reproduce

The HELPERS.md documentation indicates that you can access `generator_factory` using the following:

```
local null_ls = require("null-ls")
null_ls.helpers.generator_factory ...
```

In fact, the `null_ls` module does not have a helpers property. 
https://github.com/jose-elias-alvarez/null-ls.nvim/blob/main/lua/null-ls/init.lua

### Expected behavior

`null_ls` has a `helpers` property, or the documentation is updated to show suggested usage.

### Actual behavior

`null_ls.helpers` is nil.

### Debug log

n/a

### Help

Yes

### Implementation help

I can submit a PR but please recommend the approach you would like people to use. That is, which API do you want to expose?

**Comment by jose-elias-alvarez** [comment]
This is more of a mistake, I think – the `helpers` module is already accessible with `require("null-ls.helpers")`. Will fix the relevant documentation when I can.

**Comment by jose-elias-alvarez** [comment]
Closed by 414ed4690583315705955c80cdf52e86cfc134aa.
